src/script/components/Modals/GroupCreation/GroupCreationModal.tsx

```diff
@@ -325,6 +326,7 @@ const GroupCreationModal: React.FC<GroupCreationModalProps> = ({
       isShown={isShown}
       onClosed={onClose}
       data-uie-name="group-creation-label"
+      onKeyDown={stateIsPreferences ? handleEscape : undefined}
```

Comment:
I need to double-check here if the `Esc` keydown should be handled (and allowed), as if not - it should be prevented. Otherwise, a tab switch will happen in the main bar in the background.